README.md

```
@@ -121,4 +121,4 @@
 
 ---
 
-**Built with ❤️ for worker safety and fair employment in the Netherlands**
+**Built with ❤️ for worker safety and fair employment in the Netherlands**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)